guestbook.module

```diff
@@ -0,0 +1,177 @@
+<?php
+module_load_include('inc', 'guestbook', 'include/guestbook_db');
+module_load_include('inc', 'guestbook', 'include/guestbook_core');
+module_load_include('inc', 'guestbook', 'include/guestbook.theme');
+module_load_include('inc', 'guestbook', 'include/guestbook_weather');
+
+/**
+* @file
+* Build page, form for guestbook and weather block.
+*/
+
+/**
+* Implements hook_menu(). 
+*/
+ function guestbook_menu() {
+   $items = array();
+   // Main page.
+   $items['guestbook-page'] = array(
+     'title' => 'Guestbook',
+     'page callback'   => 'drupal_get_form',
+     'page arguments'  => array('guestbook_page_form'),
+     'access callback' => TRUE,
```

Comment:
the correct way is leave the default access callback and set the permission for your module to view this page. Then you will check this permission for anonymous users in admin UI.